docs/conan.md

```diff
@@ -0,0 +1,147 @@
+
+## Getting started
+
+1. [Install Conan](https://docs.conan.io/en/latest/installation.html)
+2. Execute in terminal: `conan profile new default --detect`
+
+## Download dependencies
+
+0. If your platform is not on the list of supported ones or you don't want to use our prebuilt binaries, you can still build dependencies from source or try consuming prebuilt binaries from the central Conan repository - [ConanCenter](https://conan.io/center/). In this case skip to the next section directly.
+
+1. Check if your build environment can use the prebuilt binaries: basically, that your compiler version (or Xcode major version) matches the information below. If you're unsure, just skip to next step.
+
+    - macOS: libraries are built with Apple clang 13 (Xcode 13.4.1), should be consumable by Xcode and Xcode CLT 13.x
+    - iOS: libraries are built with Apple clang 13 (Xcode 13.4.1), should be consumable by Xcode 13.x
+
+2. Download the binaries archive and unpack it to `~/.conan` directory:
+
+    - [macOS](https://github.com/vcmi/vcmi-deps-macos/releases/latest): pick **intel.txz** if you have Intel Mac, otherwise - **intel-cross-arm.txz**
+    - [iOS](https://github.com/vcmi/vcmi-ios-deps/releases/latest)
+
+3. Only if you have Apple Silicon Mac and trying to build for macOS or iOS: follow [instructions how to build Qt host tools for Apple Silicon](https://github.com/vcmi/vcmi-ios-deps#note-for-arm-macs), on step 3 copy them to `~/.conan/data/qt/5.15.x/_/_/package/SOME_HASH/bin` (`5.15.x` and `SOME_HASH` are placeholders).
+
+## Generate CMake integration
+
+Conan needs to generate CMake toolchain file to make dependencies available to CMake. See `CMakeDeps` and `CMakeToolchain` [in the official documentation](https://docs.conan.io/en/latest/reference/conanfile/tools/cmake.html) for details.
+
+In terminal `cd` to the VCMI source directory and run the following command. If you want to download prebuilt binaries from ConanCenter, also read the next section.
+
+<pre>
+conan install . \
+  --install-folder=<b><i>conan-generated</i></b> \
+  --no-imports \
+  --build=<b><i>never</i></b> \
+  --profile:build=default \
+  --profile:host=<b><i>CI/conan/PROFILE</i></b>
+</pre>
+
+The highlighted parts can be adjusted:
+
+- ***conan-generated***: directory where the generated files will appear. This value is used in CMake presets from VCMI, but you can actually use any directory and override it in your local presets.
+- ***never***: use this value to avoid building any dependency from source. You can also use `missing` to build recipes, that are not present in your local cache, from source.
+- ***CI/conan/PROFILE***: if you want to consume our prebuilt binaries, ***PROFILE*** must be replaced with one of filenames from our [Conan profiles directory](../CI/conan) (determining the right file should be straight-forward). Otherwise, either select one of our profiles or replace ***CI/conan/PROFILE*** with `default` (your default profile).
```

Comment:
"if you want to consume our rebuild binaries"
can you specify the cases why I may want to consume them?

"determining the right file should be straight-forward"
suggest to specify possible options, e.g. macos-arm, macos-intel

Reply:
> can you specify the cases why I may want to consume them?

there's no specific case, it's up to you to decide. For iOS you can't get binaries from ConanCenter, for example.

> suggest to specify possible options, e.g. macos-arm, macos-intel

they're given on wiki and in examples

Reply:
> For iOS you can't get binaries from ConanCenter, for example.

Don't you want to specify those scenarios?

Reply:
why should I specify them if they're clear if you open supported platforms link given in the beginning of `Using prebuilt binaries from ConanCenter` section?

another [obvious] case of not using binaries (our or from ConanCenter): you have zero trust to binaries published on the web by random people.

our deps binaries have customised options to produce the smallest possible build on CI for end users. Developers can decide what they want.